cmake/features.cmake

```diff
@@ -121,7 +121,7 @@ else()
 endif()
 
 ie_dependent_option(NGRAPH_ONNX_IMPORT_ENABLE "Enable ONNX importer" ON "protoc_available" OFF)
-ie_dependent_option(NGRAPH_ONNX_EDITOR_ENABLE "Enable ONNX Editor" ON "NGRAPH_ONNX_IMPORT_ENABLE" OFF)
+ie_dependent_option(NGRAPH_ONNX_FRONTEND_ENABLE "Enable ONNX FrontEnd" OFF "NGRAPH_ONNX_IMPORT_ENABLE" OFF)
```

Comment:
Can frontend be enabled by default everywhere when NGRAPH_ONNX_IMPORT_ENABLE is ON? This is to make sure that 'frontend' can be correctly built on all supported platforms

Reply:
Agree, don't see a reason how have multiple options about the similar stuff

Reply:
> Can frontend be enabled by default everywhere when NGRAPH_ONNX_IMPORT_ENABLE is ON?
That'd make MO to use ONNX frontend by default, which I don't think we want it yet

Reply:
In this case, maybe set name for onnx frontend as something temporary, like "onnx_experimental"? So that it will not conflict with legacy 'onnx' name in MO. I just want to be sure that there are no build issues of onnx frontend on supported platforms

Reply:
> Can frontend be enabled by default everywhere when NGRAPH_ONNX_IMPORT_ENABLE is ON? This is to make sure that 'frontend' can be correctly built on all supported platforms

The feature is currently under development. We have separate PRs for testing against CI with the feature turned ON and we'll merge them after we complete the feature. For now the feature will be turned OFF be defautlt.

Reply:
How can we merge changes when we don't know if they are even compilable?

Reply:
We manually enable `NGRAPH_ONNX_FRONTEND_ENABLE=ON` in a number of CI environments. We are just not ready to turn it on by default, especially in the context of Model Optimizer.